lib/src/utils/transaction_error_util.dart

```diff
@@ -0,0 +1,23 @@
+/// SPDX-License-Identifier: AGPL-3.0-or-later
+
+String extractTransactionErrorMessages(Map<String, dynamic> json) {
+  final messageBuffer = StringBuffer();
+
+  void extractMessages(Map<String, dynamic> map) {
+    map.forEach((key, value) {
+      if (key == 'message') {
+        messageBuffer.write(' - ');
```

Comment:
Doing this, extracted message will always start with a '-'.

Is this intended @redDwarf03 ?